### ISSUE TITLE
Long Description.

### ISSUE THREAD

**Issue by mthang** [galaxyproteomics/tools-galaxyp#713]
Is there an option to have a section only for the long description like this? This type of long description also shows up in the tool panel. 
https://github.com/galaxyproteomics/tools-galaxyp/blob/cdf3e9652b10c7a0b179202129a797e32fd95909/tools/openms/MSFraggerAdapter.xml#L6

Thanks,

**Comment by bgruening** [comment]
Which section are you referring too? You can use the tool help for important notes.

**Comment by mthang** [comment]
I am referring to line 6 to line 15 which is a section of the license agreement. A Galaxy Australia just spotted the long description appeared on the tool panel, so I am curious if there is a way to improve the user experience. I thought about it moving the long description to the "help" in line 73 (the license parameter).

**Comment by bernt-matthias** [comment]
Yeah. This is a bug of the automatic tool generation .. I thought I already fixed it 

https://github.com/OpenMS/OpenMS/issues/6568

https://github.com/bernt-matthias/CTDConverter/commit/8b245b0f0ee28dc4e49997dc01e59afc7b6a4725

**Comment by bernt-matthias** [comment]
Would it help to just update this single tool, without bumping tool version?
